deviceplugin.Dockerfile

```diff
@@ -22,7 +22,6 @@ COPY LICENSE /app/LICENSE
 RUN CGO_ENABLED=0 make device-plugin-build worker-build
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9
-RUN microdnf install -y lshw-B.02.19.2 && microdnf clean all
```

Comment:
Yes it does! I did some testing initially using `scratch` as the base image, but that would fail with the initContainer/lifecycle commands. I had forgotten about `ubi-micro` 🤦‍♂️